JqueryLibrary/src/methods/text.js

```diff
@@ -0,0 +1,9 @@
+export const text = (pointTags) => {
+    let result = '';
+
+    pointTags.forEach(item => {
+        result += (' ' + item.innerText);
```

Comment:
you can do the same with template literals 
```suggestion
        result += ` ${item.innerText}`;
```